### PR TITLE
Process tree POC with ES

### PR DESCRIPTION
**Pull request by JiaweiWu**
This PR has the changes shown in the demo (namely integration with ES)

Also merges with upstream

### CODE REVIEW

**Comment by JiaweiWu** [comment]
Also we're merging to main now right?

**Comment by opauloh** [comment]
> Also we're merging to main now right?

yes, just updated that right before the PR, to follow the upstream, let me know if find any issues with that

File: x-pack/plugins/session_view/public/components/SessionView/index.tsx [hunk 1]
```diff
@@ -46,18 +67,139 @@ export const SessionView = ({ sessionId }: SessionViewDeps) => {
...
-      <div css={processTreeCSS}>
-        <ProcessTree
-          sessionId={sessionId}
-          forward={mockData}
-          searchQuery={searchQuery}
-          selectedProcess={selectedProcess}
-          onProcessSelected={onProcessSelected}
+    <EuiPage>
+      <EuiPageContent>
+        <EuiPageHeader
```
Comment (mitodrummer) [inline x-pack/plugins/session_view/public/components/SessionView/index.tsx:175]: should this piece live in SessionViewPage?

**Review by mitodrummer — APPROVED** [review]
just the one comment. but LGTM!

**Review by opauloh — APPROVED** [review]
thanks for pushing the demo

File: x-pack/plugins/session_view/public/components/SessionView/index.tsx (hunk 1 above)
Comment (JiaweiWu) [reply x-pack/plugins/session_view/public/components/SessionView/index.tsx:175]: Yea it definitely could, although I envisioned the `SessionViewPage` to be a top level "route" page that just scaffolds whatever the main component is for that route, but yea we don't have any consensus there in terms of how to structure these things

Comment (mitodrummer) [reply x-pack/plugins/session_view/public/components/SessionView/index.tsx:175]: ya all good. I'm sure this stuff will change as we go.

**Review by zizhouW — APPROVED** [review]
LGTM!

**Pull request merged**